keyboards/zsa/voyager/rules.mk

```diff
@@ -0,0 +1,8 @@
+MCU_LDSCRIPT = voyager
+
+CUSTOM_MATRIX = lite
+DFU_ARGS = -d 3297:0791 -a 0 -s 0x08002000:leave
+DFU_SUFFIX_ARGS = -v 3297 -p 0791
```

Comment:
So it looks like the bootloader actually emulates DfuSe, not just generic DFU (like `stm32duino` does)?

Supporting this bootloader in QMK Toolbox would need a separate PR (and rerunning the driver installer on Windows, unless the bootloader implements Microsoft USB descriptors to install the WinUSB driver automatically).  Also if you would make different revisions of the bootloader, note that QMK Toolbox uses only the VID:PID pair to determine the flashing command, therefore if you ever need to change the firmware start address, you would need to allocate a new PID for that bootloader variant (and update QMK Toolbox again).

Reply:
Noted, and thanks.

Reply:
Microsoft USB descriptors are used. And VID:PID change for address change, noted.